### ISSUE TITLE
MySQL Error

### ISSUE THREAD

**Issue by ShadowRanger** [LogBlock/LogBlock#610]
I seem to be getting this MySQL error every now and then. I'm not sure exactly what causes it but I thought it might be worth reporting.

http://pastebin.com/9zVVdKuG

**Comment by LefsFlare** [comment]
This might not be it but try this SQL statement,

``ALTER TABLE `lb-world-chest` MODIFY COLUMN `itemdate` INT NOT NULL``

and see if it fixes your problem ^_^

**Comment by frymaster** [comment]
Can you tell me:

a) what the result of ``/version logblock`` is, and
b) What the value of the "version" field is in the logblock config?

**Comment by ShadowRanger** [comment]
a) LogBlock v1.94-dev-SNAPSHOT (build #255)
b) 1.94

I'm always keeping up to date with the latest versions. I'm pretty sure this error has only started popping up with these latest versions although I'm not 100% sure.

**Comment by frymaster** [comment]
Yeah, if you can set that to 1.80 and let me know what happens please

(there's a decent chance it will error out, in which case set it back to 1.94 manually)

Does this happen every time you move a completed potion (ie not just a bottle of water) to or from a chest?

tl;dr the itemdata column used to be TINYINT (0-255) and is now SMALLINT (0-65535) in order to accommodate item durability and potion data.  The database should automatically be updated when  you update logblock (because it notices that the config says the DB is old)

The next thing to try would be to download a slightly older build (number 250) which had better diagnostics (later builds moved to a more efficient database system but it doesn't provide the same level of debug info)

**Comment by frymaster** [comment]
Great
